### ISSUE TITLE
VA-11 Hall-A

### ISSUE THREAD

**Issue by koukichi7010** [skyline-emu/skyline-games-list#92]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Tested Build

#1058

### Android Version

11

### SoC type

Snapdragon

### Device GSMArena link

https://www.google.com/amp/s/m.gsmarena.com/realme_6_pro-ampp-10114.php

### Game Behaviour

![Screenshot_2022-04-29-07-03-56-99_fcf072b9e4efc4ce5ca3e36691a7c3c6](https://user-images.githubusercontent.com/86339491/165861769-a354faf8-00e9-4952-b581-eb80bf33025f.jpg)
Basically just deadlock, it won't boot

### Is Log Level is set to 'Info'?

- [X] Yes

### Log file

[It's in #Support on Discord (Click Me) or the one below](https://discord.com/channels/545842171459272705/960961451340754954/969373694617067521) 

https://discord.com/channels/545842171459272705/960961451340754954/969373694617067521
